### PR TITLE
Iis prod

### PR DESCRIPTION
**Pull request by jhackettpps**

### CODE REVIEW

File: azure-prod-common.tf [hunk 1]
```diff
@@ -0,0 +1,31 @@
+variable "azure_subscription_id" {
```
Comment (mattops) [inline azure-prod-common.tf]: Is this file mean to be a symlink?

Comment (jhackettpps) [reply azure-prod-common.tf]: yep

Comment (jhackettpps) [reply azure-prod-common.tf]: actually, no: this file is in the root of the repo and is intended to replace azure-prod.tf in the same way as azure-devtest-common.tf replaces azure-devtest.tf

Comment (mattops) [reply azure-prod-common.tf]: Ok cool.

**Pull request merged**